pkgs/games/maptool/default.nix

```diff
@@ -76,6 +73,7 @@ stdenvNoCC.mkDerivation {
   inherit pname version src meta;
 
   dontUnpack = true;
+  dontConfigure = true;
   dontBuild = true;
   dontWrapGApps = true;
```

Comment:
Does any of those do an unwanted change? Otherwise please keep them enabled.

Reply:
`dontUnpack` fails the build if omitted, because the source file is detected as an archive but the default builder can't unpack it.

`dontWrapGApps` keeps an unnecessary wrapper layer from being added; when we make our own wrapper, we include the `gappsWrapperArgs` there.

`dontConfigure` and `dontBuild` keep the builder from printing
```
configuring
no configure script, doing nothing
building
no Makefile or custom buildPhase, doing nothing
```
which is just unhelpful noise. The construction of this package is such that it'd be nonsensical to hook or override the configure or build phases; there is no upside to including them.